packages/core/src/index.ts

```diff
@@ -9,17 +9,15 @@ export * from "./context/AgenticaTokenUsage";
 export * from "./events/AgenticaCallEvent";
 export * from "./events/AgenticaCancelEvent";
 export * from "./events/AgenticaDescribeEvent";
-// EVENTS
 export * from "./events/AgenticaEvent";
 export * from "./events/AgenticaEventSource";
-
 export * from "./events/AgenticaExecuteEvent";
 export * from "./events/AgenticaRequestEvent";
 export * from "./events/AgenticaResponseEvent";
 export * from "./events/AgenticaSelectEvent";
 export * from "./events/AgenticaTextEvent";
+export * as factory from "./factory";
```

Comment:
Personally, I think it would be better if the factory name were broken down in the factory/index.ts file.

The term "factory" is a bit too broad and abstract, so I’d like to propose splitting it into more specific modules like eventFactory, operationFactory, and promptFactory.

While the amount of logic in operation might be small, I believe it should still be grouped for the sake of consistency.